exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/FileSystemConfig.java

```diff
@@ -53,18 +53,23 @@ public class FileSystemConfig extends StoragePluginConfig {
   public static final String NAME = "file";
 
   private final String connection;
+  private final String[] mountCommand, unmountCommand;
```

Comment:
Is there a reason we're using `String[]` and not an `ArrayList` here?

Reply:
@cgivre apologies, I missed this. The motivation was runtime efficiency since I don't need the overhead of dynamic collections here. The Runtime.exec method that I hand off to is also based on String[] so by matching that I don't need to do a List to array conversion. Are Lists preferable when working with arrays from Jackson though, because I can certainly change this?

Reply:
After a little more thought your suggestion won me over and I've converted them to Lists.